packages/ui/src/machines/authenticator/defaultServices.ts

```diff
@@ -14,7 +17,28 @@ export const defaultServices = {
   async getAmplifyConfig() {
     const result = Amplify.getConfig();
     groupLog('+++getAmplifyConfig', 'result', result);
-    return result;
+
+    const cliConfig = (result as ResourcesConfig).Auth.Cognito;
```

Comment:
Curious why we need to cast the return type of `getConfig`?

Reply:
Good call, this was just when I copy pasted from the other method as it needed the casting. Here it does not since it's already ResourcesConfig